src/app/GitCommands/Settings/AppSettings.cs

```diff
@@ -722,6 +722,41 @@ private static void MigrateSshSettings()
             }
         }
 
+        /// <summary>
+        /// Migrate editor settings if GE was installed in 'Program Files (x86)' before 4.3.
+        /// Only check and update global Git settings in Windows, the only set by GE.
+        /// Guess previous, migrate to current path (i.e. the first usage).
+        /// </summary>
+        private static void MigrateEditorSettings()
+        {
+            // Only run this once
+            bool isMigrated = IsEditorSettingsMigrated;
+            if (isMigrated)
```

Comment:
```suggestion
            if (IsEditorSettingsMigrated)
```

Reply:
Missed to add back the migration check!

```suggestion
            bool isMigrated = IsEditorSettingsMigrated;
            IsEditorSettingsMigrated = true;
            if (isMigrated)
```